### PR TITLE
DOCS-350: Clarification about BGP Peer parameters

### PR DESCRIPTION
**Pull request by girishshankaran**
This change is applicable to other versions of the OSS docs. Once this note is reviewed and approved, I'll go ahead and propagate the change into the other versions as well.

Product Version(s):

Issue:
(https://tigera.atlassian.net/browse/DOCS-350)

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

**Comment by girishshankaran** [comment]
@nelljerram Can you please review this one?

File: calico/reference/resources/bgppeer.mdx [hunk 1]
```diff
@@ -60,6 +60,12 @@ the [node resource](node.mdx).
 
 :::
 
+:::note
+
+If you want to use the non-default BGP Peer parameters, you must disable node-to-node mesh by setting the `nodeToNodeMeshEnabled` parameter to false in the [BGPConfiguration](./bgpconfig) resource.
```
Comment (nelljerram) [inline calico/reference/resources/bgppeer.mdx]: Can I suggest instead "If you want to use BGPPeer resources to configure the BGP sessions between nodes within the cluster - for example, in order to set non-default BGPPeer parameters - you should disable the node-to-node mesh ..."

**Review by nelljerram — APPROVED** [review]
LGTM, thanks Girish.

**Comment by ctauchen** [comment]
LGTM @girishshankaran

**Pull request merged**